lib/Support/ModuleInterpreter.cpp

```diff
@@ -361,6 +365,13 @@ void ModuleInterpreter::invoke(bool express_type) {
   case mem_mode_t::PART_TENSOR_IN_MEM:
     invoke_part_in_mem(express_type);
     break;
+  case mem_mode_t::ALL_TENSOR_IN_DISK:
+    if(FILE *file = fopen("./value2disk.npz", "rb")){
+        fclose(file);
+        std::remove("./value2disk.npz");
+      }
+    invoke_to_disk("./value2disk.npz",express_type);
```

Comment:
This hard coding file name "value2disk.npz" could be improved using llvm TempFile or a related strategy.

Reply:
In the new commit, the llvm TempFile class was used, which seems to be more complex. In order to obtain the file on the Python side, an additional pybind interface was added because the file name was randomly generated.